### ISSUE TITLE
OAuthd: Integrating a Google App as a provider

### ISSUE THREAD

**Issue by kalanbates** [oauth-io/oauthd#133]
TL;DR
Can someone please manually confirm that Google can be added as an auth provider inside an oauthd app? I'm having some trouble getting it added to my oauthd instance.

---

I'm having some difficulty getting a Google provider set up in my oauthd instance.  I'm coming here because I'm hoping your team might have expertise with setting up a google provider appropriately in oauthd.

To give an idea of where I am in my process:
I've already gotten a facebook provider configured, integrated, and working like a champ with oauthd and I've leveraged the angular module provided for the [oauth-io/oauth-js] sdk to perform a full end-to-end authentication test using oauthd.  So, with that working, I'm trying to add google as an alternate provider to my web application.

I went to the Google developer's console and created a new Project.
Inside that project, under APIs & auth | Credentials | OAuth, I clicked "Create new Client ID."  For the sake of discussion, let's assume that the Client ID that gets generated is "abc123.apps.googleusercontent.com" and that the Client Secret that gets generated is "f00bar"
I specified a Redirect URI identical to the one I registered with my facebook app.  
I added the same Javascript Origins that I had whitelisted inside my facebook app.

Inside my oauthd app, I added Google as a new provider, and placed the generated "Client ID" (abc123.apps.googleusercontent.com) from the google app into the provider registration's "client_id" field, and placed the generated "Client Secret"(f00bar) from the Google app into the provider registration's "client_secret" field.

I then hit Save this provider.
Back at the application portal, I hit the "Try auth" button associated with the Google provider, and I get a popup that attempts to call out to google, but it cannot load the page.  

It is requesting to load https://accounts.google.com/o/oauth2/auth with the following queryString parameters: client_id=myclientId, response_type=code, redirect_uri=http://auth.mydomain.com:6284/auth, state=ZXpcMLXjbHg1aB0-yjqZYTVeBb4

Is there anything that I'm missing here?  I did notice that the client_secret is not being sent down in the token request, but I'm not in control of the arguments that oauthd sends when "Try Auth" is triggered.

Thanks.
